### DISCUSSION TITLE
V8.0.0Beta

### DISCUSSION THREAD

**Discussion by liu21st** [top-think/framework#2827] [Announcements]
基于PHP8的重构版本
架构不变

<hr /><em>This discussion was created from the release <a href='https://github.com/top-think/framework/releases/tag/v8.0.0-beta'>V8.0.0Beta</a>.</em>

**Comment by big-dream** [comment]
扬帆 起航

**Comment by WongYueKeung** [comment]
多应用可以支持开启think-swoole 吗？用tp6的时候我一直认为这个是个遗憾的地方，希望tp8能够重整雄风 支持使用think-swoole扩展后开启多应用模式。多应用是很重要的，应用之间可以共享很多common的东西，很大的程度上能够提升项目的开发效率和可维护，属于thinkphp的一个很大的独有特色。
用了think-swoole之后app maping，自动解析域名到app，共享配置文件的功能都没了，我觉得算是个要重点攻克的地方，感激不尽
